packages/core/src/modules/ledger/LedgerModule.ts

```diff
@@ -26,47 +27,58 @@ export class LedgerModule {
     await this.ledgerService.connectToPools()
   }
 
-  public async registerPublicDid(did: string, verkey: string, alias: string, role?: NymRole) {
```

Comment:
If we add the TAA to the indy pool config, there's no need to specify it for each method we call. We can extract it from the indy pool config at a lower level